app/graphql/resolvers/business/query.js

```diff
@@ -1,9 +1,14 @@
+import RuralPaymentsPortal from '../../../services/rural-payments-portal.js'
+import RuralPaymentsPortalBusinessTransformer from '../../../transformers/rural-payments-portal/business.js'
 import { Land } from '../land/query.js'
 
+const ruralPaymentsPortal = new RuralPaymentsPortal()
```

Comment:
Just a note for when you come to refactor. This is fine for now — we know there are plans to change up how this is working based on the `RESTDataSource` — but this class could still be injected via the server's context object, as this is a good pattern to follow generally.

See https://www.apollographql.com/docs/apollo-server/data/fetching-rest#adding-data-sources-to-your-servers-context-function

Reply:
noted, will work this into the refactor